config.toml

```
@@ -0,0 +1,6 @@
+baseURL = "https://unbalancedparentheses.github.io/compiler-team/"
+languageCode = "en-us"
+title = "Rust Lang - Compiler Team"
+
```

The theme that you're using supports having a "last modified by" author/date populated by Git information, I think this would be useful to have:

```suggestion

enableGitInfo = true
```